src/meta/processors/usersMan/AuthenticationProcessor.cpp

```diff
@@ -112,27 +118,66 @@ void DropUserProcessor::process(const cpp2::DropUserReq& req) {
 
 
 void GrantProcessor::process(const cpp2::GrantRoleReq& req) {
-    const auto& roleItem = req.get_role_item();
-    CHECK_SPACE_ID_AND_RETURN(roleItem.get_space_id());
-    CHECK_USER_ID_AND_RETURN(roleItem.get_user_id());
     folly::SharedMutex::WriteHolder wHolder(LockUtils::userLock());
+    GraphSpaceID spaceId;
+    const auto& roleItem = req.get_role_item();
+    auto userRet = getUserId(roleItem.get_user());
+    if (!userRet.ok()) {
+        handleErrorCode(MetaCommon::to(userRet.status()));
+        onFinished();
+        return;
+    }
+    if (roleItem.get_role_type() == nebula::cpp2::RoleType::GOD) {
+        spaceId = kDefaultSpaceId;
```

Comment:
Do we support to grant a user to be GOD?

Reply:
> Do we support to grant a user to be GOD?

Only supported to grant a user to GOD in initialization phase. 
We need to initialize a root user before  the authority flag turn on.